backend/de.metas.frontend-testing/src/main/java/de/metas/frontend_testing/masterdata/CreateMasterdataCommand.java

```diff
@@ -79,6 +80,7 @@ public JsonCreateMasterdataResponse execute()
 		this.context = new MasterdataContext();
 
 		// IMPORTANT: the order is very important
+		createResources();
```

Comment:
pls use the same pattern and methods that we use for the other master data creation